doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -9,22 +9,93 @@
+* Private (cluster internal network)
+* Storage
+
+The network topology should provide at least 2 networks with preallocated IP addressing schemes for the out-of-band management network and the Public network. The Public network must be able to reach / access the public Internet, preferably directly or via an enterprise proxy. At least IPv4 addresses must be available, ideally IPv6 address space should also be supported. At least 1 network interface must be capable of performing PXE boot and that network must be available to both the jump / test host and each physical server.
+
+The network fabric shall comprise redundant layer 2 connectivity between hosts. The switching hardware shall support VLANs for separating networks as well as supporting line speeds corresponding to the speeds of the NICs listed below. 
+
 
 <a name="3.3"></a>
 ## 3.3 Infrastructure Requirements
 
-> Describe the requirements on any infrastructure used to deploy the RI onto (i.e. from the HW profile and SW profile specifications in RA2).
+The requirements on infrastructure hardware used to deploy and run the RIi2 are as follows:
+
+1. One (1) physical server dedicated as a jump / test host
+1. Six (6) physical servers, serving as either controller or compute/worker nodes
+
+The jump / test host allows to install and manage the operation of the RI2. For example, the host can be used to execute the installer software to deploy and configure the RI2 on the other physical servers. Once installed, the host may be used to run test tools or it may be used to generate test traffic intended for performance measurements. To support these operations, the physical server provided for the jump / test host shall meet the same hardware requirements as the controller / compute hosts, as outlined below. In this context, this host may be considered as a miscellaneous use or general purpose system.
+
+Each physical server shall meet the following minimum specifications:
+
+* CPU
+  *  2x x86_64 CPU sockets, providing 20 cores each, 40 simultaneous multi-threads (SMT), at 2.2 GHz
+* Memory
+  * 256 GB RAM
+* Storage
+  * 1.0 TB SSD via SATA 6 Gbps
+* Network Interfaces
+  * 4x 10 Gbps Ethernet Ports
+  * Out-of-band Management Port
+
+
 
 <a name="3.4"></a>
 ## 3.4 Software Requirements
 
-> Describe any requirements for running the installer software and the validation scripts.
+The following section describes requirements on the installer software of the RI2 and the validation scripts.
+
+The overall installation process should be logically separated in a node provisioning and a Kubernetes installation phase. This serves the purpose to allow deployment of the RI2 on different infrastructures, such as bare-metal private cloud deployments or on public cloud infrastructure providers. By logically separating the installation, the same Kubernetes deployment tooling can be re-used across a variety of infrastructures, thereby enabling re-use of RI2 components.
```

Comment:
I don't think the reference here, "...separated in a node provisioning", is to kubernetes node. Maybe change to "... separated in a host provisioning"

Reply:
done